suite-native/module-send/src/components/SendForm.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import {
+    formInputsMaxLength,
+    SendFormValues,
+    sendFormValidationSchema,
+} from '@suite-common/validators';
+import { VStack, Button, Text } from '@suite-native/atoms';
+import { useForm, Form, TextInputField } from '@suite-native/forms';
+import { AccountKey } from '@suite-common/wallet-types';
+import {
+    AccountsRootState,
+    FeesRootState,
+    selectAccountByKey,
+    selectNetworkFeeInfo,
+} from '@suite-common/wallet-core';
+
+type SendFormProps = {
+    accountKey: AccountKey;
+};
+
+export const SendForm = ({ accountKey }: SendFormProps) => {
+    const account = useSelector((state: AccountsRootState) =>
+        selectAccountByKey(state, accountKey),
+    );
+
+    const networkFeeInfo = useSelector((state: FeesRootState) =>
+        selectNetworkFeeInfo(state, account?.symbol),
+    );
+
+    const form = useForm<SendFormValues>({
+        validation: sendFormValidationSchema,
+        context: {
+            networkFeeInfo,
+            networkSymbol: account?.symbol,
+            availableAccountBalance: account?.availableBalance,
+        },
+        defaultValues: {
+            address: '',
+            amount: '',
+        },
+    });
+
+    const {
+        handleSubmit,
+        formState: { isValid },
+    } = form;
+
+    const handleValidateForm = handleSubmit(() => {
+        // TODO: start on-device inputs validation via TrezorConnect
+    });
+
+    const amountTransformer = (value: string) =>
+        value
+            .replace(/[^0-9\.]/g, '') // remove all non-numeric characters
+            .replace(/(?<=\..*)\./g, '') // keep only first appearance of the '.' symbol
```

Comment:
what if users uses decimal `,`  do we take this into account?

Reply:
, will be removed same as desktop does. Not sure if it is idea, but let's not take care of perfect formatting until it is decided by product.